src/components/structures/InteractiveAuth.js

```diff
@@ -140,19 +140,21 @@ export default React.createClass({
         });
     },
 
-    _requestCallback: function(auth) {
+    _requestCallback: function(auth, background) {
         this.setState({
-            busy: true,
+            busy: !background,
```

Comment:
Actually this was a bit silly (and buggy). I've updated it so it doesn't set the state at all for background requests, and commented it.